README.md

```diff
@@ -0,0 +1,41 @@
+# dc-chartreuse
+
+_an open source project by [wh](https://github.com/wh0),_
+_README provided by [Hiroyuki](https://github.com/WeebHiroyuki)._
+
+## What is dc-chartreuse?
+
+dc-chartreuse is a prototype relay system for making Discord bots usable and viable on serverless-hosting platforms such as [Glitch](https://glitch.com) and [Heroku](https://heroku.com).
+
+**Disclaimer:** This is a general introduction and more simplified version of this project. If you would like to read more details and information about this project, feel free to read the [blog post](https://support.glitch.com/t/a-prototype-bot-relay-for-discord/27845) posted by wh. It contains an in-depth explanation of this project, its purpose, some background information concerning the project, as well as the conventions utilized in the process of achieving the project’s goal.
+
+## Introduction to the Project and FAQs (Frequently Asked Questions)
+
+### Why did wh create this project?
+
+Due to the nature and behavior of the [Discord API](https://discord.com/developers/docs/intro), hosting your Discord bot on a serverless hosting provider such as Glitch or Heroku becomes quite difficult.
+Heroku and Glitch’s “serverless” design causes complications for developers trying to host and run their Discord bot.
+Basically, the Discord API uses a gateway technique. This requires a constant stable connection to be open 24/7 for the bot to operate. This conflicts with Heroku and Glitch’s design because as mentioned previously, it’s serverless, which means that operations will only be performed when a request is made to the server. Once the server becomes inactive again, it will be shut down.
+With that being said, wh created this for developers to be able to circumvent Glitch and Heroku’s serverless system by maintaining an active gateway connection.
+
+### A security vulnerability may be involved — a man in the middle.
```

Comment:
I probably shouldn't have used the phrase "man in the middle," because it's so commonly associated with an attack. It's a component that forwards data, and that's not especially a security vulnerability compared to other parts of a system design, such as a hosting provider.

Reply:
Alright. Uh, how is the term "relayer" as a replacement?

Reply:
Maybe "relay"? I think it can just be used as a noun. I'll change it later if I come up with a more specific word for it.

Reply:
Sure, that's a good idea. Let me know if there are any other issues with the pull request.